feat(cli): Note that you can use sh (#6058)
getsentry/sentry-docs: src/docs/product/cli/installation.mdx

This will automatically download the correct version of `sentry-cli` for your operating system and install it. If necessary, it will prompt for your admin password for `sudo`. For a different installation location or for systems without `sudo` (like Windows), you can `export INSTALL_DIR=/custom/installation/path` before running this command.

> <Note>
> 
> You may also wish to use `sh` over `bash` if you are in a constrained environment.
> 
> </Note>

Reviewer: ```suggestion
<Note>

If you're in a constrained environment, you may wish to use `sh` over `bash`.

</Note>
```
